nifi-nar-bundles/nifi-evtx-bundle/nifi-evtx-processors/src/test/java/org/apache/nifi/processors/evtx/ParseEvtxTest.java

```diff
@@ -366,7 +366,7 @@ public void fileGranularityLifecycleTest() throws IOException, ParserConfigurati
         assertEquals(1, failureFlowFiles.size());
         validateFlowFiles(failureFlowFiles);
         // We expect the same number of records to come out no matter the granularity
-        assertEquals(960, validateFlowFiles(failureFlowFiles));
+        assertEquals(1053, validateFlowFiles(failureFlowFiles));
```

Comment:
Overall LGTM, but can we have a bit more talkative constants to avoid using magic numbers like this multiple times?